contrib/pg_tde/src/keyring/keyring_kmip.c

```diff
@@ -204,3 +206,15 @@ get_key_by_name(GenericKeyring *keyring, const char *key_name, KeyringReturnCode
 
 	return key;
 }
+
+static void
+validate(GenericKeyring *keyring)
```

Comment:
There is a ping command in the protcol but I do not think our library supports it.

Reply:
also in this case at least establishing the connection should verify that the keys we have are okay.

Reply:
Yeah, I think this is fine. If that library had supported the ping command it would have been worth using, but now: no.